src/backend/access/aocs/aocsam_handler.c

```diff
@@ -1858,6 +1868,7 @@ aoco_index_build_range_scan(Relation heapRelation,
 	 * only one of those is requested.
 	 */
 	Assert(!(anyvisible && checking_uniqueness));
```

Comment:
We can replace `checking_uniqueness` with `(indexInfo->ii_Unique || indexInfo->ii_ExclusionOps != NULL)`. In this case we can remove variable and ifdef